samplers/jaegerremote/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package jaegerremote implements the Jaeger Remote protocol.
+package jaegerremote // import "go.opentelemetry.io/contrib/samplers/jaegerremote"
```

Comment:
what is the purpose of the `// import` comment? If it's part of the documentation, then perhaps it should be on the line above?

Reply:
It is a [canonical import path](https://go.dev/doc/go1.4#canonicalimports) and belongs inline after the package name.

Reply:
+1